common/metrics/dashboard_components.go

```diff
@@ -42,18 +42,18 @@ var dashboardClient *dashboard.DashboardsClient
 
 // MonitoringMetricsResources contains information required to create the monitoring dashboard
 type MonitoringMetricsResources struct {
-	ProjectId                string
-	DataflowJobId            string
-	DatastreamId             string
-	GcsBucketId              string
-	PubsubSubscriptionId     string
-	SpannerInstanceId        string
-	SpannerDatabaseId        string
-	ShardToDataStreamNameMap map[string]string
-	ShardToDataflowInfoMap   map[string]internal.ShardedDataflowJobResources
-	ShardToPubsubIdMap       map[string]internal.PubsubCfg
-	ShardToGcsMap            map[string]internal.GcsResources
-	ShardId                  string
+	ProjectId                     string
+	DataflowJobId                 string
+	DatastreamId                  string
+	MetadataGcsBucketId           string
```

Comment:
Maybe this is not super clear to me but isn't the metadata gcs bucket the TmpDir passed by users? This isn't the same gcs bucket. This refers to the gcs bucket associated with each shard

Reply:
Naming things is hard I guess, they are both Metadata bucket in a sense, but this bucket is more _meta_ than the one used in Datastream since that one actually contains customer data, while this one will persist HB local stuff.